src/lib/components/Skeleton/Skeleton.svelte

```diff
@@ -1,14 +1,24 @@
 <script lang="ts">
   import { onMount } from 'svelte';
-
+  import { fade } from 'svelte/transition';
   let mounted = false;
   onMount(async () => {
     mounted = true;
   });
 </script>
 
 {#if mounted}
-  <slot name="mounted" />
+  <div in:fade={{ duration: 500 }}>
```

Comment:
If you want **every** wrapped skeleton-wrapped component to fade-in, you could do that. Otherwise, move this effect to the caller :point_down: at `routes/index.svelte`